include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -184,7 +245,8 @@ namespace depthimage_to_laserscan
       int offset = (int)(cam_model.cy()-scan_height/2);
       depth_row += offset*row_step; // Offset to center of image
 
-      for(int v = offset; v < offset+scan_height_; v++, depth_row += row_step){
+      for(int v = offset; v < offset+scan_height_; v++, depth_row += row_step)
+      {
```

Comment:
Please avoid whitespace changes only to keep merges as easy as possible. I won't object to the newlines being inserted, but there are a few other lines with this rewrapping of unchanged lines which makes the review harder.